#goal definition
float32 desired_x
float32 desired_y
---
#result definition
bool success
---
#feedback
float32 current_x
float32 current_y

